src/events.ts

```diff
@@ -0,0 +1,35 @@
+import * as SDL from "../mod.SDL.ts";
```

Comment:
Don't import from `mod.SDL.ts` from `src`.

Suggestion:
```ts
import { Event } from "./SDL/events.ts";
import { GetError, WaitEvent } from "./SDL/functions.ts";
```

You'll need to update the rest of the code then as well.